apps/src/p5lab/spritelab/commands/storyLabCommands.js

```diff
@@ -20,5 +20,17 @@ export const commands = {
 
   setSubheading(subheading) {
     this.storyLabText.subheading = subheading;
+  },
+
+  getHeading(heading) {
```

Comment:
nit: Do these methods need arguments (`getHeading` and `getSubheading`)?

Reply:
Ha, they do not! Thank you for catching. (I copied and pasted the lines above to create these and tested that they work, but missed this!